id/proguard-rules.pro

```diff
@@ -52,6 +52,21 @@
 -keep class com.simprints.infra.events.event.domain.models.** { *; }
 -keep class com.simprints.infra.events.event.local.** { *; }
 
+-keep class com.simprints.feature.orchestrator.steps.** { *; }
```

Comment:
Wouldn't this completely disable obfuscation for those feature modules? Maybe we could have something more specific - e.g. everything in those modules that end in Result?

Reply:
In general yes. However, I wonder if it's worth the effort!? As the code is open now, there's no benefit to obfuscation besides a very small reduction in binary size. Given the amount of bugs that obfuscation causes I'd even argue that disabling obfuscation for the whole project may be a good idea.